src/plugins/augeas/augeas.c

```diff
@@ -216,9 +216,6 @@ static int foreachAugeasNode (augeas * handle, const char * treePath, ForeachAug
 
 		result = (*callback) (handle, curPath, callbackData);
 
-		/* handle the subtree */
-		result = foreachAugeasNode (handle, curPath, callback, callbackData);
```

Comment:
Ok, nice, thank you. Is this also faster or otherwise unrelated to the benchmark?